client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-definitions/setup/index.tsx

```diff
@@ -1,4 +1,5 @@
 import { Task } from '@automattic/launchpad';
+import { isBlogOnboardingFlow } from '@automattic/onboarding';
 import { addQueryArgs } from '@wordpress/url';
 import { recordTaskClickTracksEvent } from '../../tracking';
 import { TaskAction } from '../../types';
```

Comment:
Nit: I think this should be a type import.

Reply:
Fixed https://github.com/Automattic/wp-calypso/pull/86584/commits/ed8db385028cd43eeb3a4d39369dee1b0c9d1c20

Reply:
Fixed here https://github.com/Automattic/wp-calypso/pull/86584/commits/511e0cebb0f7e9d536dafc34e4494dec6d5f251f